apps/meteor/app/models/server/models/Users.js

```diff
@@ -1006,6 +1006,7 @@ export class Users extends Base {
 			{
 				active: true,
 				federated: true,
+				username: { $regex: /.*:.*/ },
```

Comment:
do we really need to match the username as well? I mean, isn't `federated: true` enough?

performing a regex is not the most efficient thing

Reply:
No, revisiting this again, it's not necessary, tks.